exchanges/okgroup/okgroup_types.go

```diff
@@ -1409,110 +1389,30 @@ type WebsocketUserSwapPositionHoldingData struct {
-	// Leverage    A member, but part already exists as part of WebsocketDataResponse
-	// Price       A member, but part already exists as part of WebsocketDataResponse
-	// Type        A member, but part already exists as part of WebsocketDataResponse
-}
-
-// WebsocketUserFuturePositionResponse contains formatted data for futures positions data
-type WebsocketUserFuturePositionResponse struct {
-	LongQty              string `json:"long_qty"`
-	LongAvailQty         int64  `json:"long_avail_qty"`
-	LongAvgCost          string `json:"long_avg_cost"`
-	LongSettlementPrice  string `json:"long_settlement_price"`
-	RealisedPnl          string `json:"realised_pnl"`
-	ShortQty             string `json:"short_qty"`
-	ShortAvailQty        string `json:"short_avail_qty"`
-	ShortAvgCost         string `json:"short_avg_cost"`
-	ShortSettlementPrice string `json:"short_settlement_price"`
-	LiquidationPrice     string `json:"liquidation_price"`
-	Leverage             string `json:"leverage"`
-	CreatedAt            string `json:"created_at"`
-	UpdatedAt            string `json:"updated_at"`
-	LongMargin           string `json:"long_margin"`
-	LongLiquiPrice       string `json:"long_liqui_price"`
-	LongPnlRatio         string `json:"long_pnl_ratio"`
-	ShortMargin          string `json:"short_margin"`
-	ShortLiquiPrice      string `json:"short_liqui_price"`
-	ShortPnlRatio        string `json:"short_pnl_ratio"`
-	LongLeverage         string `json:"long_leverage"`
-	ShortLeverage        string `json:"short_leverage"`
-	// UpdatedAt          A member, but part already exists as part of WebsocketDataResponse
-	// MarginMode         A member, but part already exists as part of WebsocketDataResponse
-	// InstrumentID A member, but part already exists as part of WebsocketDataResponse
-}
-
 // WebsocketSpotOrderResponse contains formatted data for spot user orders
 type WebsocketSpotOrderResponse struct {
-	FilledNotional float64 `json:"filled_notional,string"`
-	FilledSize     float64 `json:"filled_size,string"`
-	Notional       float64 `json:"notional,string"`
-	Size           float64 `json:"size,string"`
-	Status         string  `json:"status"`
-	MarginTrading  int64   `json:"margin_trading,omitempty"`
-	Type           string  `json:"type"`
-	// Price        A member, but part already exists as part of WebsocketDataResponse
-	// InstrumentID A member, but part already exists as part of WebsocketDataResponse
-	// Timestamp    A member, but part already exists as part of WebsocketDataResponse
-	// OrderID      A member, but part already exists as part of WebsocketDataResponse
+	Table string `json:"table"`
+	Data  []struct {
+		ClientOid      string    `json:"client_oid"`
+		CreatedAt      string    `json:"created_at"`
+		FilledNotional string    `json:"filled_notional"`
+		FilledSize     string    `json:"filled_size"`
+		InstrumentID   string    `json:"instrument_id"`
+		LastFillPx     string    `json:"last_fill_px"`
+		LastFillQty    string    `json:"last_fill_qty"`
```

Comment:
Many of these should be converted to more useful types instead of autoconverted to strings